apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/HistoryConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.openhouse.jobs.util;
+
+import com.linkedin.openhouse.tables.client.model.Retention;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+/** History config class. This is app side representation of /tables policies->history */
+@Builder
+@Getter
+@EqualsAndHashCode
+@ToString
+public class HistoryConfig {
```

Comment:
why do we need this class as part of this change?

Reply:
I'll remove this, I moved this change to the jobs (Snapshot Expiration) change